pages/09.contacts/01.manage-contacts/02.contact-monitoring/docs.en.md

```diff
@@ -149,7 +149,9 @@ Currently, `utm_medium`, `utm_source`, `utm_campaign`, `utm_content`, and `utm_t
 
 All the Utm tags are available in the time entry, just by toggling the entry details button.
 
-Please note that UTM tags are recorded only on a form submission that contains the action "Record UTM Tags".
+Mautic automatically tracks UTM tags for all requests (including visitors).
+
+Before version 3.2, UTM tags were only recorded on a form submission that contains the action "Record UTM Tags".
```

Comment:
```suggestion
Before [Mautic 3.3][mautic-3.3], UTM tags were only recorded on a form submission that contains the action "Record UTM Tags".
```

Reply:
Please add the link macro at the bottom of the page: 

`[mautic-3.3]: <https://github.com/mautic/mautic/releases/tag/3.3>`